src/components/DigitalAssets/Asset/Asset.js

```diff
@@ -1,103 +1,165 @@
-      <div className='tick'>
-        {/* @TODO: no size prop in JIcon */}
-        <JIcon name='checkbox' color={(color === 'white') ? 'blue' : 'white'} />
-      </div>
-      <div className='symbol'>
-        <JAssetSymbol symbol={symbol} color={assetColor} />
-      </div>
-      <div className='name'>
-        <JText value={name} color={assetColor} weight='bold' />
-      </div>
-      {!isPopular && (
-        <div className='balance'>
-          <AssetBalance
-            symbol={symbol}
-            color={assetColor}
-            balance={balance}
-            isLoading={isLoading}
-          />
-        </div>
-      )}
-      {isCustom && (
-        <div className='edit'>
-          <JFlatButton
-            onClick={ignoreEvent(edit)(address)}
-            iconSize='small'
-            iconColor='gray'
-            iconName='settings'
-            isTransparent
-          />
-        </div>
-      )}
-    </div>
-  )
-}
-
-type Props = {
-  edit: Function,
-  hover: Function,
-  setActive: Function,
-  name: string,
-  symbol: string,
-  address: Address,
-  color: 'blue' | 'white',
-  balance: number,
-  isCustom: boolean,
-  isActive: boolean,
-  isLoading: boolean,
+type AssetsState = {
   isHovered: boolean,
-  isPopular: boolean,
 }
 
-AssetCard.defaultProps = {
-  edit: () => {},
-  hover: () => {},
-  setActive: () => {},
-  name: ethereum.name,
-  symbol: ethereum.symbol,
-  address: ethereum.address,
-  color: 'white',
-  balance: 0,
-  isCustom: false,
-  isActive: false,
-  isLoading: false,
-  isHovered: false,
-  isPopular: false,
+class AssetCard extends PureComponent<Props, AssetsState> {
+  static defaultProps = {
+    name: ethereum.name,
+    symbol: ethereum.symbol,
+    address: ethereum.address,
+    fiatMoney: 'USD',
+    balance: 0,
+    fiatBalance: 0,
+    isLoading: false,
+    isLongLoading: false,
+    isHovered: false,
+  }
+
+  constructor(props: Props) {
+    super(props)
+
+    this.state = {
+      isHovered: false,
+    }
+  }
+
+  onHover = (isHovered: boolean) => {
+    this.setState({ isHovered })
+  }
+
+  render() {
+    const {
+      name,
+      symbol,
+      address,
+      fiatMoney,
+      balance,
+      fiatBalance,
+      isLoading,
+      isLongLoading,
+    } = this.props
+
+    const {
+      isHovered,
+    } = this.state
+
+    const isLoadingContext = isLoading ? 'loading' : isLongLoading ? 'long-loading' : null
+
+    return (
+      <div
+        onMouseEnter={handle(this.onHover)(true)}
+        onMouseLeave={handle(this.onHover)(false)}
+        className={classNames('asset-card')}
+      >
+        <div className='symbol'>
+          <JAssetSymbol symbol={symbol} color='gray' />
+        </div>
+        <div className='name'>
+          {/*
+            @TODO: consultation with the programmer on the size of the text:
+            size 'header' has insufficient size
+          */}
+          <JText value={name} color='dark' weight='bold' size='header' />
+        </div>
+
```

Comment:
we don't need to add useless empty lines in jsx. We need to add eslint rule for this